src/modules/land_detector/VtolLandDetector.cpp

```diff
@@ -62,17 +63,35 @@ void VtolLandDetector::_initialize_topics()
 	MulticopterLandDetector::_initialize_topics();
 
 	_airspeedSub = orb_subscribe(ORB_ID(airspeed));
+	_vehicle_status_sub = orb_subscribe(ORB_ID(vehicle_status));
 }
 
 void VtolLandDetector::_update_topics()
 {
 	MulticopterLandDetector::_update_topics();
 
 	_orb_update(ORB_ID(airspeed), _airspeedSub, &_airspeed);
+	_orb_update(ORB_ID(vehicle_status), _vehicle_status_sub, &_vehicle_status);
+}
+
+bool VtolLandDetector::_get_maybe_landed_state()
+{
+
+	// Only trigger in RW mode
+	if (!_vehicle_status.is_rotary_wing) {
+		return false;
+	}
+
+	return MulticopterLandDetector::_get_maybe_landed_state();
 }
 
 bool VtolLandDetector::_get_landed_state()
 {
+	// Only trigger in RW mode
+	if (!_vehicle_status.is_rotary_wing) {
```

Comment:
There is no land detector for FW mode at the moment. Ground testing would only be affected at the level of auto disarm.

If we want full land detection also when the vtol is in fw this will need to be completely refactored.